CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.14] - 2025-04-10
+
+### Added
+- asLeader/asFollower utilities for distributed Skip services (#853).
+- Various Skiplang/Skargo improvements for native code (#838-#847, #849)
```

Comment:
My take so far has been to only mention changes that directly impact users (therefore none of the `skiplang` stuff). What do you think?

Reply:
Sounds fine to me, I'll drop that line.